packages/shared/src/pluginStore.ts

```diff
@@ -3,78 +3,74 @@ import type {
   PluginStore,
   RsbuildPlugin,
   RsbuildPluginAPI,
-  BundlerPluginInstance,
+  RsbuildPlugins,
 } from './types';
 import { color, isFunction } from './utils';
 
-function validatePlugin(plugin: unknown) {
```

Comment:
why remove the validatePlugin function

Reply:
I removed it while resolving conflicts. Should I bring it back?